client/verta/tests/http_requests/test_connection.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+
+from verta._internal_utils import credentials
+from verta._internal_utils._utils import Connection, _GRPC_PREFIX
+
+
+https_scheme = "https"
+fake_socket = "test:8080"
+custom_headers = {"custom-header": "custom-header-value"}
```

Comment:
Just out of curiosity, was there a reason for making these lowercase rather than uppercase as constants?